Resources/config/services.xml

```diff
@@ -18,34 +19,34 @@
 
     <services>
 
-        <service id="libphonenumber.phone_number_util" class="%libphonenumber.phone_number_util.class%"/>
+        <service id="libphonenumber.phone_number_util" class="libphonenumber\PhoneNumberUtil"/>
```

Comment:
If you use the actual class name here instead of the container parameter, you could remove the parameters altogether. They don't have any effects now. But note that this breaks applications for people relying on the old behaviour (changing the parameter values now doesn't change anything at all).